code/argv_list.py

```diff
@@ -1,2 +1,4 @@
+#!/usr/bin/env python
```

Comment:
Including the shebang in files would require us taking time to explain what this is to learners. Since it is not already in the lesson material, we would need to justify the time it would require to add to a workshop's limited time frame. I see it as being difficult to justify including the shebang in the context of the workshop's limited time and scope, since it is not required to include in a script.

Reply:
Yes, I know, the only reason I decided to include this part is because there was a file like this already.
I think it's either all of them have shebangs or none to make the whole codebase consistent.
If you think that's too much I can remove all of them and update the draft.

Reply:
@maxim-belkin do you have a preference on including or removing all shebangs when making these consistent?